.travis.yml

```diff
@@ -42,11 +42,13 @@ matrix:
 
 before_install:
 - npm config set spin false
-- npm install -g coveralls pr-bumper@^1.0.0
+- npm install -g bower
```

Comment:
Why is this necessary when Bower is listed as a _devDependency_?  Are we going to have to add it to other repos too that don't currently have this entry?

Reply:
The build was failing and I am in the middle of determining if I need to `npm install -g bower` and do `bower install` in travis CI or just run `bower install`

Reply:
@notmessenger - Alright, the build is passing now with just running `bower install` in travis.